charts/pulsar/values.yaml

```diff
@@ -609,14 +615,14 @@ broker:
   # use a component name that matches your grafana configuration
   # so the metrics are correctly rendered in grafana dashboard
   component: broker
-  replicaCount: 3
+  replicaCount: 2
```

Comment:
keep this setting as default?

Reply:
@sijie , does the new commits satisfy your request?